api/src/main/java/edu/cornell/mannlib/vedit/beans/DynamicFieldRow.java

```diff
@@ -34,4 +35,13 @@ public void setParameterMap(Map parameterMap) {
         this.parameterMap = parameterMap;
     }
 
+    public String getLanguage() {
+        return language;
+    }
+
+    public void setLanguage(String language) {
+        if (language != null) {
```

Comment:
This if clause is a little bit strange here. I understand that null value might include some issue later, but I think you used StringUtils.isNotBlank wherever you invoked this method. I believe this might be removed.